fedmsg_meta_fedora_infrastructure/greenwave.py

```diff
@@ -32,17 +32,17 @@ class GreenwaveProcessor(BaseProcessor):
 
     @staticmethod
     def satisfied(msg):
-        try:
-            # https://pagure.io/greenwave/pull-request/100
```

Comment:
We can't remove this, since this code needs to remain working with all previously sent messages.

Reply:
Yeah, agreed.